pages/content/amp-dev/documentation/tools.html

```diff
@@ -61,7 +61,7 @@ <h2 class="ap-o-stage-content-subline">for creation, design and development</h2>
 <amp-state id="activeFilter">
```

Comment:
That's a nit but can we rename "activeFilter" to just "filter" and "chosenFilter" to "format"?

Reply:
okaay, done :)